WordPress/Tests/PHP/EnqueuedCheckUnitTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Unit test class for WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Tests\PHP;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+/**
+ * Unit test class for the EnqueuedCheck sniff.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   1.0.0
+ */
+class EnqueuedCheckUnitTest extends AbstractSniffUnitTest {
+
+	/**
+	 * Returns the lines where errors should occur.
+	 *
+	 * @return array <int line number> => <int number of errors>
+	 */
+	public function getErrorList() {
+		return array(
+			3 => 1,
+			8 => 1,
+			9 => 1,
+			10 => 1,
+			22 => 1,
+			25 => 1,
+			28 => 1,
+		);
+	}
+
+	/**
+	 * Returns the lines where warnings should occur.
+	 *
+	 * @return array <int line number> => <int number of warnings>
+	 */
+	public function getWarningList() {
+		return array(
+			3  => 1,
+			16 => 1,
+			17 => 1,
+			18 => 1,
+			19 => 1,
+			22 => 1,
+		);
+	}
+
+} // End class.
```

Comment:
We've recently removed all `// End` comments in the WPCS code. Please remove them from this PR as well.